packages/schema/src/config/common.ts

```diff
@@ -134,10 +134,14 @@ export default defineUntypedSchema({
    */
   modulesDir: {
     $default: ['node_modules'],
-    $resolve: async (val, get) => [
-      ...await Promise.all(val.map(async (dir: string) => resolve(await get('rootDir'), dir))),
-      resolve(process.cwd(), 'node_modules')
-    ]
+    $resolve: async (val, get) => {
+      const rootDir = await get('rootDir')
+      const currentDir = process.cwd()
```

Comment:
I don't think this needs to be split out; it's not async or used more than once.

Reply:
Oh you're right, my bad. I thought for a second that it was part of the promise, I apologize!